docs/content/posts/2024-03-21-release-kubernetes-operator-1.8.0.md

```diff
@@ -0,0 +1,159 @@
+---
+title:  "Apache Flink Kubernetes Operator 1.8.0 Release Announcement"
+date: "2024-03-21T18:00:00.000Z"
+authors:
+- mxm:
+  name: "Maximilian Michels"
+  twitter: "stadtlegende"
+- gyfora:
+  name: "Gyula Fora"
+  twitter: "GyulaFora"
+- 1996fanrui:
+  name: "Rui Fan"
+  twitter: "1996fanrui"
+aliases:
+- /news/2024/03/21/release-kubernetes-operator-1.8.0.html
+---
+
+The Apache Flink community is excited to announce the release of Flink Kubernetes Operator 1.8.0!
+
+The release includes many improvements to the operator core, the autoscaler, and introduces new features
+like TaskManager memory auto-tuning.
+
+We encourage you to [download the release](https://flink.apache.org/downloads.html) and share your experience with the
+community through the Flink [mailing lists](https://flink.apache.org/community.html#mailing-lists) or
+[JIRA](https://issues.apache.org/jira/browse/flink)! We're looking forward to your feedback!
+
+## Highlights
+
+### Flink Autotuning
```

Comment:
Should we call this `Flink Memory Autotuning` ?

Reply:
I see you have reference of other config tuning in the future, it is also okay to keep it like this :)

Reply:
I had thought about that as well but decided to market it as a more general feature which hosts various functionality.